src/game.zig

```diff
@@ -680,7 +709,7 @@ pub fn gamemodeToggle() void {
 
 
 pub fn update(deltaTime: f64) void { // MARK: update()
-	const gravity = 30.0;
+	const gravity = 30.0 * (1 - @round(collision.isPlayerInFluid(Player.super.pos, Player.outerBoundingBox) * 100) / 100);
```

Comment:
Why are you rounding the value here?
Also you shouldn't change gravity itself, as this also unintentionally impacts other processes like in this case reducing air resistance.
I would prefer if you would just add the buoyancy to the acceleration z below.